core/src/accounts_hash_verifier.rs

```diff
@@ -83,6 +85,21 @@ impl AccountsHashVerifier {
         fault_injection_rate_slots: u64,
         snapshot_interval_slots: u64,
     ) {
+        let mut time = Measure::start("hash");
+
+        let simple_capitalization_enabled = true; // ??? TODO
```

Comment:
could we just forward this through in the `AccountsPackage` struct?

Reply:
Yes. I wasn’t sure which structs could be changed and if that was better or to look it up somehow.

Reply:
doh. turns out we don't need it anyway. We are not interested in calculating lamports anyway.